documentation

carasomov opened the pull request.

carasomov committed:
README.md

Anastasia-Mishalova commented on README.md:
```diff
@@ -14,3 +14,44 @@
 А теперь самое интересное – вопросы! В **Quizzify Quester** вопросы делятся на темы. Каждая тема требует своего названия, давайте назовем их, например, "Котики", "Кино", "Космос" и так далее. Создавайте их сколь угодно, дайте им название и едем дальше.
 
 Дали название теме? Тогда переходим к созданию вопросов! Вопрос может содержать текстовое описание или изображение. А единственный верный ответ на вопрос также может быть представлен ввиде текста или изображения — это тот момент, когда ваше знание, ну или "Я УГАДАЮ" приводят к победе, или поражению.... Количесво вопросов в каждой из тем также может быть любым. 
```
" — это тот момент, когда ваше знание, ну или "Я УГАДАЮ" приводят к победе, или поражению...." 
Можно удалить, завершив предложение на слове "изображения"

Anastasia-Mishalova commented on README.md:
```diff
@@ -14,3 +14,44 @@
 
 Дали название теме? Тогда переходим к созданию вопросов! Вопрос может содержать текстовое описание или изображение. А единственный верный ответ на вопрос также может быть представлен ввиде текста или изображения — это тот момент, когда ваше знание, ну или "Я УГАДАЮ" приводят к победе, или поражению.... Количесво вопросов в каждой из тем также может быть любым. 
+
+Поздравляю!Теперь вы научились создавать викторину на свой цвет и вкус,но начинать играть еще рано.Вопросы - важно,но так же важны и правила
```
"на свой вкус и цвет", это устойчивое выражение, употребляемое именно в таком порядке слов

Anastasia-Mishalova commented on README.md:
```diff
@@ -14,3 +14,44 @@
+
+# Правила
+
+После того как ведущий создал викторину и пригласил игроков.Первое что необходимо сделать команде - взять с собой позитивный настрой и немного свободного времени.Если все это у вас есть,то ведущий(он же создатель викторины) выбирает случайного игрока.Перед игроком есть таблица вопросами и темой,на клетках в вопросами есть их стоимость в очках.Когда игрок выбрал клетку с вопросом,отображается сам  ...
```
После знаков препинания ( . , ! ? : ...) всегда ставится пробел (кроме определенных случаев, их можно загуглить)

Anastasia-Mishalova replied:
Опечатка в слове "кропку", дальше по тексту тоже есть опечатки, все указывать не буду, но смысл понят. Перед отправкой на ревью надо проходиться по всему тексту еще раз, чтобы исключить ошибки по невнимательности

Anastasia-Mishalova commented on README.md:
```diff
@@ -14,3 +14,44 @@
+На вопрос отвечает тот кто среди игроков первым нажал на "ПРОБЕЛ"
+В случии когда игрок оказался прав в своем ответе,то игроку на баланс начисляются столько очков,во сколько оценивается данный вопрос.В случии проигрыша очки отнимаются
+В этом случии два пути "на вопроск осталось время" и **"время истекло"**
+в первом случии выходит отвечать другой игрок.А вв случии когда времени на таймере нет,вопрос закрывается,после чего игрокам показывается правильный ответ и ведущий уже выбирает другого игрока для выбора вопроса
```
Также стоит обращать внимание на повторения. Тут словосочетание "в случае" повторяется пять раз в пяти предложениях подряд, так делать не надо

Anastasia-Mishalova commented on README.md:
```diff
@@ -14,3 +14,44 @@
+
+Игрок оказался прав? Супер! Ему на счет прибавляется стоимость вопроса в очках и остальным игрокам показывается правильный ответ, после чего ведущий выбирает другого игрока. Хочется дать совет, не гуглите ответы на вопросы, в викторине важен ни сколько правильный ответ,сколько впечатления и азарт игроков
+
+Но что если игрок ошибься, тогда он получает 0? Из баланса игрока вычитается стоимость вопроса в очка.Ограничений нет и вы с самого начала игры можете,так сказать "уйти в долг"
```
Ошибка в слове "ошибся". Лайфхак - можно вставить весь свой текст в ворд, допущенные в словах ошибки там будут подсвечиваться и их можно будет легко и быстро исправить

Anastasia-Mishalova reviewed (commented):
**Один большой общий комментарий** (Также некоторые отдельные моменты я указала ниже, они применимы в принципе к любому тексту)

Текст правил должен быть чуть более кратким и более "официальным".
То есть, без "...которые сейчас происходят в каждом утюге", "Конечно,мы бережем ваше время", "...если вы облактились на пробел и случайно вызвались отвечать на вопрос,то не вините нас". Всё должно быть максимально понятно и просто.
Попробуй использовать ИИ. Просто вбей в поисковик "чатгпт онлайн", выбери любой сайт и попробуй сформулировать запросы с правилами для нашей игры (чем подробнее ты опишешь запрос, тем лучше будет результат от ИИ). Обычно искусственный интеллект выдает довольно хороший результат без лишней воды и заигрывания с пользователем.

Обязательно отредактируй полученные ответы от ИИ под нашу игру, так как даже с самым подробным запросом он может обосраться и что-то  упустить. Это уже будет хорошая база для правил.

Если не будет что-то получаться или будут вопросы по тексту из разряда "норм-не норм", можешь не стесняться и писать мне в ЛС дискорда или телеграма

carasomov committed:
Update README.md

carasomov committed:
2Update README.md

carasomov committed:
UpREADME.md

Немного почистил от воды,убрал фразы по типу "из каждого утюга" "Помните: мир, дружба, жвачка"
